src/utilities/getObjectValue.ts

```diff
@@ -0,0 +1,19 @@
+ /**
+ * Get value for object from path.
+ * @param {string} path - Property name of object. For nested objects use property.nested
+ */
+export function getObjectValue(theObject: any, path: string, separator = '.') {
```

Comment:
how's the performance of this function? try it with 400 000 elements vs 400 000 elements in the old version just to be on the safe side.

Reply:
I've done some testing, both components on 400-500k rows work almost equally fast